src/main/daml/Daml/Finance/Instrument/Bond/FloatingRate/Instrument.daml

```diff
@@ -39,9 +41,9 @@ template Instrument
       -- ^ The instrument's version.
     description : Text
       -- ^ A description of the instrument.
-    referenceRateId : Text
-      -- ^ The floating rate reference ID. For example, in case of "3M Euribor + 0.5%" this should
-      --   be a valid reference to the "3M Euribor" reference rate.
+    floatingRate : BondTypes.FloatingRate
+      -- ^ A description of the floating rate to be used. This supports both Libor and SOFR style
```

Comment:
Not exactly true, as we support only SOFR index and not the general calculation methodologies. 

Shall we say ... Libor and compounded rate indexes?

Reply:
I think the index methods corresponds to one of the three complete calculation variants. I rephrased it to clarify.